.github/workflows/publish-documentation-next.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
           persist-credentials: false
-      - uses: grafana/writers-toolkit/publish-technical-documentation@publish-technical-documentation/v1 # zizmor: ignore[unpinned-uses]
```

Comment:
You should be able to allow these pins now since Renovate understands tags with slash separators: https://github.com/renovatebot/renovate/pull/35431

https://github.com/grafana/writers-toolkit/commits/9aed4f7c5b3caa1b58d72ee39232333407d4acbf is referenced by `publish-technical-documentation/v1`.

Reply:
Nice! It looks like Renovate automatically swapped it over on its own here. I'll update this to remove the zizmor ignore.

That regex tho! 😵‍💫